pkg/secretless/config/v2/service.go

```diff
@@ -186,6 +188,60 @@ func connectorID(
 	return connectorID, nil
 }
 
+// ValidateConfig is a function object that validates the connectors requested
+// against the plugins available. Its static dependencies are a logger and the
+// plugins available to create connectors.
+type ValidateConfig struct {
+	logger logapi.Logger
+	availPlugins plugin.AvailablePlugins
+}
+
+// NewValidateConfig creates a new instance of ValidateConfig.
+func NewValidateConfig(lg logapi.Logger, ap plugin.AvailablePlugins) ValidateConfig {
+	return ValidateConfig{
+		logger: lg,
+		availPlugins: ap,
+	}
+}
+
+// Call validates that the connectors of the given Config exist, according to
+// the AvailablePlugins.
+func (v *ValidateConfig) Call(cfg Config) error {
```

Comment:
Not a blocker but we don't really use the `Call` pattern almost anywhere else and it doesn't describe what the method is doing to the params.